examples/apollo-server/index.ts

```diff
@@ -24,13 +24,11 @@ const getEnveloped = envelop({
 const server = new ApolloServer({
   schema,
   executor: async requestContext => {
-    const { schema, execute, contextFactory } = getEnveloped({ req: requestContext.request.http });
+    const { perform } = getEnveloped({ req: requestContext.request.http });
 
-    return execute({
-      schema,
-      document: requestContext.document,
-      contextValue: await contextFactory(),
-      variableValues: requestContext.request.variables,
+    return perform({
+      query: requestContext.document,
```

Comment:
Not sure about this one because Apollo already runs `validate`, and this will make it called twice, no?

Reply:
Ah ok, well, in that case - we should use `execute` instead.

Reply:
Or maybe disable the validation in Apollo? 🤔

Reply:
We cannot disable parse and validate in Apollo's Server; so, using `getEnveloped.execute` is the way to go here.